core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -451,12 +451,12 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       withLibraryAttributeNamespaceCheck(libraryAttributeNames) {
         dataSource.inTransaction { dataAccess =>
           withWorkspaceContextAndPermissions(sourceWorkspaceName, WorkspaceAccessLevels.Read, dataAccess) { sourceWorkspaceContext =>
-            withClonedRealm(sourceWorkspaceContext, destWorkspaceRequest) { newRealm =>
+            withClonedAuthDomain(sourceWorkspaceContext, destWorkspaceRequest) { newAuthDomain =>
```

Comment:
name misleading here, should be withClonedAuthDomains (plural) and newAuthDomains